.nvmrc

```diff
@@ -0,0 +1 @@
+22 
```

Comment:
```suggestion
22.0.0
```

Reply:
otherwise nvm can't find the version!

Reply:
@dielduarte I made it `lts/jod` -- does that work on your machine? when i do 22.0.0, it uses that specific version, rather than the most recent compatible with 22.x.x